use-cases/azure.md

```diff
@@ -0,0 +1,319 @@
+```
+### Start RabbitMQ docker container
+
+You can now **start the RabbitMQ docker container** based on the image you just built.
+
+When starting it, you need to:
+- Mount the `rabbitmq.config` file to the right location
+- Give the `rabbitmq.config` file proper permissions (the UID of rabbitmq user in the image we are going to use is **999**)
+- Set the `RABBITMQ_ADVANCED_CONFIG_FILE` environment variable to tell RabbitMQ where to find the `rabbitmq.config` file
+- Expose the RabbitMQ Management UI port (**15672**)
+
+```
+chown 999:999 ./rabbitmq.config
+docker run -p 15672:15672 \
+           -e RABBITMQ_ADVANCED_CONFIG_FILE=/etc/rabbitmq/conf.d/rabbitmq.config \
+           -v /root/rabbitmq.config:/etc/rabbitmq/conf.d/rabbitmq.config \
+           -d pivotalrabbitmq/rabbitmq:3.10.5-26.g6a3825f
+```
+
+## Start Reverse Proxy
+
+### Generate SSL certificate and key
+
+> :warning: Remember when you have registered your app on Azure AD that it only allows **https** protocol for OAuth2 **Redirect URI**? That's why we are going to deploy RabbitMQ Management UI behind a nginx Reverse Proxy, which is going to act as TLS Termination Endpoint (feel free to use another tool, or something already set-up at your place, if you prefer)
+
+For the purpose of this tutorial, you can generate a self-signed certificate/key pair (answer the questions and enter the Common Name when prompted):
+```
+openssl req -newkey rsa:2048 -nodes -keyout KEY_FILENAME.key -x509 -days 365 -out CERT_FILENAME.crt
+```
+
+### Create proxy configuration
+
+Update the reverse proxy configuration file [proxy.conf](../conf/azure/proxy.conf):
+```
+$ vi proxy.conf
+server {
+        listen 443 ssl;
+        server_name     PUT_YOUR_RABBITMQ_HOSTNAME;
+ 
+        ssl_certificate      ./PUT_YOUR_CERT_FILENAME.crt;
+        ssl_certificate_key  ./PUT_YOUR_KEY_FILENAME.key;
+ 
+        ssl_session_cache    shared:SSL:10m;
+        ssl_session_timeout  5m;
+ 
+        ssl_protocols TLSv1.2 TLSv1.1 TLSv1 ;
+        ssl_ciphers  EECDH+AESGCM:EDH+AESGCM:EECDH+AES256:EDH+AES256:ECDH+AES128:DH+AES:ECDH+3DES:DH+3DES:RSA+AESGCM:RSA+AES:RSA+3DES:!aNULL:!MD5;
+        ssl_prefer_server_ciphers   on;
+ 
+  location / {
+ 
+     proxy_pass http://PUT_YOUR_RABBITMQ_HOSTNAME:15672;
+     proxy_set_header Host $http_host;
+     proxy_set_header X-Forwarded-Proto $scheme;
+     proxy_set_header X-Real-IP $remote_addr;
+     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+  }
+}
+```
+> :arrow_right: This file will be loaded in nginx to configure it as a reverse proxy for our RabbitMQ Management UI.
+
+### Run nginx container
+
+We can now run our `nginx` container, without forget to:
+- Mount `proxy.conf`, `CERT_FILENAME.crt` and `KEY_FILENAME.key` in `/etc/nginx/conf.d`
+- Expose HTTPS port **443**
+
+```
+docker run -d -p 443:443 -v ./proxy.conf:/etc/nginx/conf.d/proxy.conf -v ./rabbit.crt:/etc/nginx/rabbit.crt -v ./rabbit.key:/etc/nginx/rabbit.key nginx:latest
```

Comment:
To make it dead easy for community using this tutorial, I was thinking of providing scripts and invoke those scripts from a target in the Makefile.
For instance, what about a target in the Makefile  called `start-azure-proxy` ? Up to you if you prefer to call directly docker from the Makefile target or move the docker statement to a separate script under `bin/azure` whichever way is easy. Sometimes i find it easier to do the latter.